src/bacnet/basic/service/h_ts.c

```diff
@@ -276,3 +291,13 @@ void handler_timesync_init(void)
     }
 }
 #endif
+
+void handler_timesync_set_callback_set(
+    handler_timesync_set_callback_t cb)
+{
+    if (cb) {
```

Comment:
There is no need to check the cb for NULL if we are going to set the callback to NULL when the cb is NULL.

Reply:
Thanks. Fixed in 898923b9e0f690f3dc63f5ea0c1ee7d8316a041e